cvat-ui/src/components/invitations-page/invitations-list.tsx

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2023 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React, { useCallback, useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Row, Col } from 'antd/lib/grid';
+import Pagination from 'antd/lib/pagination';
+
+import { CombinedState, InvitationsQuery } from 'reducers';
+import { Invitation } from 'cvat-core/src/organization';
+import { acceptInvitationAsync, rejectInvitationAsync } from 'actions/invitations-actions';
+import { updateHistoryFromQuery } from 'components/resource-sorting-filtering';
+import { useHistory } from 'react-router';
+import notification from 'antd/lib/notification';
+import InvitationItem from './invitation-item';
+
+import dimensions from '../projects-page/dimensions';
+
+interface Props {
+    query: InvitationsQuery;
+}
+
+const PAGE_SIZE = 9;
+
+function setUpInvitationsList(invitations: Invitation[], newPage: number): Invitation[] {
+    return invitations.slice((newPage - 1) * PAGE_SIZE, newPage * PAGE_SIZE);
+}
+
+export default function InvitationsListComponent(props: Props): JSX.Element {
+    const { query } = props;
+
+    const dispatch = useDispatch();
+    const history = useHistory();
+    const invitations = useSelector((state: CombinedState) => state.invitations.invitations);
+    const [page, setPage] = useState(query.page);
+    const renderedInvitations = setUpInvitationsList(invitations, page);
```

Comment:
Could you please explain? In reducer (GET_INVITATIONS_SUCCESS) I see it just assigns new invitations list, not append to existing one. So, it looks like only one page is stored in the storage.